api/v1/package/viewsets.py

```diff
@@ -118,20 +118,20 @@ def create(self, request, *args, **kwargs):
             'id': serializer.data['id'],
             'statusCode': status.HTTP_201_CREATED}, status=status.HTTP_201_CREATED)
 
-    def handle_exception(self, exc):
-        # Call the default exception handler first to get the standard error response
-        response = super().handle_exception(exc)
+    # def handle_exception(self, exc):
```

Comment:
??

Reply:
need to verify function, will check and uncomment later.